src/lib/components/Input.svelte

```diff
@@ -1,24 +1,21 @@
 <script lang="ts">
+  // props for the input
   export let type = 'text';
   export let name: string;
-  export let label: string;
   export let placeholder: string | undefined = undefined;
   export let required = false;
   export let id = '';
   export let value = '';
-  export let autocomplete = false;
+  export let autocomplete = '';
+  export let inputClass = 'mt-1 focus:ring-indigo-500 focus:border-indigo-500 block w-full shadow-sm sm:text-sm border-gray-300 rounded-md';
```

Comment:
Theres no need for this prop neither. Input's look and feel must be predictable and be the same on all instances.

```suggestion
```